ui/app/pages/home/home.component.js

```diff
@@ -125,6 +152,25 @@ export default class Home extends PureComponent {
                         key="home-backupApprovalNotice"
                       />,
                     },
+                    {
+                      shouldBeRendered: threeBoxLastUpdated && restoredFromThreeBox === null,
```

Comment:
Conditions like this might leave us in a strange state if the flag is turned on, then off again.

This situation wouldn't occur unless someone enabled it in the dev console of course, but do you think it's worth adding the flag here to prevent it anyway? It's possible an end-user might enable 3Box in the dev console then disable it, and get in a weird state like this. Maybe unlikely though - I'm OK with ignoring this case if you'd prefer. If you did want to handle it, there are a few other examples around like this as well.

Reply:
Good point. Done in 4fcc18a77